truffle.js

```diff
@@ -12,16 +12,18 @@ module.exports = {
     ropsten: {
       network_id: 3, // Official Ethereum test network
       host: 'localhost',
-      port: 8545,
+      port: 8645,
+//      from: '0x11591D432d9Ff7cF517819ba99B3D876719019Dd', // Christoph's test ether
```

Comment:
I think we should have only one address resolver for all developers. If you want to be able deploy manually we can send you keys we use for test networks.